packages/vm/Cargo.toml

```diff
@@ -64,7 +59,7 @@ bitflags = "1.1.0" # https://github.com/CensoredUsername/dynasm-rs/pull/74
 # wasmer-middlewares = { path = "../../../wasmer/lib/middlewares" }
 
 [dev-dependencies]
-criterion = { version = "0.4", features = [ "html_reports" ] }
+criterion = { version = "0.4", features = ["html_reports"] }
```

Comment:
Where do you get all those toml formatting changes from? Looks like some tool. Let's ensure we use the same.

Reply:
I believe this is caused by the "Even Better TOML" vscode extension. I was using "Better TOML", but that's no longer maintained.